CHANGELOG.asciidoc

```diff
@@ -35,6 +35,10 @@ This release also includes changes from <<release-3-3-6, 3.3.6>>.
 * Added easier to understand exceptions for connection problems in the Gremlin.Net driver.
 * Support configuring the type registry builder for GraphBinary.
 * Release working buffers in case of failure for GraphBinary.
+* GraphBinary: Use the same `ByteBuf` instance to write during serialization. Changed signature of write methods in type
+serializers.
```

Comment:
nit: no need to multi-line the changelog. it can just run on as needed on a single line.